apps/foundation-demo/src/app/checkout/actions/checkout.actions.ts

```diff
@@ -20,9 +21,18 @@ export class ShowReviewView implements Action {
 
 export class PlaceOrder implements Action {
   readonly type = CheckoutActionTypes.PlaceOrderAction;
+
+  constructor(public payload: number) {}
```

Comment:
Why do you need a number here? I'm not sure that we'll ever have a scenario where we have more than 1 cart at a time, right?

Reply:
I just added it so you couldn't just route to `/thank-you` without a particular number. It also makes it pretty difficult to get to the thank-you page from anywhere but the checkout page. Do you think there's a better way to do this?

Reply:
Removed the parameter